zendesk.json

```diff
@@ -1387,34 +1499,14 @@
-                    "data_type": "numeric",
-                    "example_value": 360011869194
-                },
-                {
-                    "data_path": "action_result.data.*.custom_fields.*.value",
-                    "data_type": "string",
-                    "example_value": "custom field value"
                 }
             ],
             "versions": "EQ(*)"
         },
         {
             "action": "update ticket",
             "description": "Update ticket information",
-            "verbose": "Update an already existing ticket with the values that are specified in the 'fields' parameter. The user has to know the key names to set in this parameter. Please refer to the Zendesk Core API, Tickets section to learn about the fields that can be used to update or create tickets.<br>The JSON that is specified in the 'fields' parameter should have the keys and values specified in double quotes string format, except in case of boolean values, which should be either <i>true</i> or <i>false</i>.<br>Some examples: <ul><li>To close a ticket: {\"subject\": \"Zeus, multiple action need to be taken\", \"status\": \"solved\"}</li><li>To add a comment: {\"status\": \"open\", \"comment\": { \"body\": \"The smoke is very colorful.\", \"author_id\": 1 }}</li><li>To update custom fields value: {\"custom_fields\": [{\"Test Field\": \"test field value\"}, {\"custom test field\": \"custom test field value\"}]}</li></ul>",
+            "verbose": "Update an already existing ticket with the values that are specified in the 'fields' parameter. The user has to know the key names to set in this parameter. Please refer to the Zendesk Core API, Tickets section to learn about the fields that can be used to update or create tickets.<br>The JSON that is specified in the 'fields' parameter should have the keys and values specified in double quotes string format, except in case of boolean values, which should be either <i>true</i> or <i>false</i>.<br>Some examples: <ul><li>To close a ticket: {\"subject\": \"Zeus, multiple action need to be taken\", \"status\": \"solved\"}</li><li>To add a comment: {\"status\": \"open\", \"comment\": { \"body\": \"The smoke is very colorful.\", \"author_id\": 1 }}</li><li>To update custom fields value: {\"custom_fields\": [{\"Test Field\": \"test field value\"}, {\"custom test field\": \"custom test field value\"}]}</li></ul>If you want to add <b>double quote(\")</b> or <b>backslash(\\)</b> in custom fields, use the <b>escape(\\)</b> character followed by double quote or backslash, check the following examples,<br><ul><li>Too add double quote or backslash in custom field values: {\"custom_fields\": [{\"Test Field\": \"test \\\"field\\\" value\"}, {\"custom test field\": \"custom \\\\test\\\\ field value\"}]}</li><li>Too add double quote or backslash in custom field keys: {\"custom_fields\": [{\"Test \\\"Name\\\" Field\": \"test field value\"}, {\"custom \\\\test\\\\ field\": \"custom test field value\"}]}</li></ul>",
```

Comment:
```suggestion
            "verbose": "Update an already existing ticket with the values that are specified in the 'fields' parameter. The user has to know the key names to set in this parameter. Please refer to the Zendesk Core API, Tickets section to learn about the fields that can be used to update or create tickets.<br>The JSON that is specified in the 'fields' parameter should have the keys and values specified in double quotes string format, except in case of boolean values, which should be either <i>true</i> or <i>false</i>.<br>Some examples: <ul><li>To close a ticket: {\"subject\": \"Zeus, multiple action need to be taken\", \"status\": \"solved\"}</li><li>To add a comment: {\"status\": \"open\", \"comment\": { \"body\": \"The smoke is very colorful.\", \"author_id\": 1 }}</li><li>To update custom fields value: {\"custom_fields\": [{\"Test Field\": \"test field value\"}, {\"custom test field\": \"custom test field value\"}]}</li></ul>If you want to add <b>double quote(\")</b> or <b>backslash(\\)</b> in custom fields, use the <b>escape(\\)</b> character followed by double quote or backslash, check the following examples,<br><ul><li>To add double quote or backslash in custom field values: {\"custom_fields\": [{\"Test Field\": \"test \\\"field\\\" value\"}, {\"custom test field\": \"custom \\\\test\\\\ field value\"}]}</li><li>To add double quote or backslash in custom field keys: {\"custom_fields\": [{\"Test \\\"Name\\\" Field\": \"test field value\"}, {\"custom \\\\test\\\\ field\": \"custom test field value\"}]}</li></ul>",